typescript/formsApps.d.ts

```diff
@@ -45,6 +45,10 @@ export type FormsAppScheduledTasksMenuItem = FormsAppBaseMenuItem & {
   tasks: Array<{ taskId: number }>
   /** If true, menu item will be the default item shown */
   isDefault: boolean
+  /** The ids of the task groups assigned to the app */
+  taskGroups: Array<{ taskGroupIds: number }>
```

Comment:
I'm not sure if this enough, because it's not the group, but an instance of it.

Reply:
ahhh you're right, it'll need a label at least too